app/models/concerns/legal_aid_application_state_machine.rb

```diff
@@ -48,6 +48,7 @@ def summary_state
         transitions from: :delegated_functions_used, to: :checking_passported_answers
         transitions from: :client_details_answers_checked, to: :checking_passported_answers
         transitions from: :provider_assessing_means, to: :checking_passported_answers
+        transitions from: :provider_checked_citizens_means_answers, to: :provider_checking_citizens_means_answers
```

Comment:
Should this transition from provider_checking_citizens_means_answers to provider_checked_citizens_means_answers, instead of the other way around as it is here?

Reply:
Thanks for noticing! We're going to the check_passported_answers page instead of the means_summaries page. I'm on the fix now.

Reply:
Removed now thanks.